sql/wsrep_applier.cc

```diff
@@ -93,12 +107,14 @@ void wsrep_store_error(const THD *const thd, wsrep::mutable_buffer &dst) {
 
   auto da = thd->get_stmt_da();
   if (da->cond_count() == 0 && da->is_set()) {
+    uint const err_code = wsrep_errno_for_voting(da->mysql_errno());
+    const char *const err_str = da->message_text();
```

Comment:
no reason to change da->message_text() and keep it as is (just pass da->message_text() as a parameter)

Reply:
Below we've got very similar construction in `for` loop. I made this two blocks of code to look similar for easier analysis.